src/main/java/io/openapiprocessor/maven/ProcessMojo.java

```diff
@@ -28,21 +29,18 @@ public class ProcessMojo extends AbstractMojo {
     @Parameter(required = true)
     private String id;
 
-    @Parameter(required = false)
-    private String processorName;
-
-    @Parameter(required = false)
+    @Parameter(required = true)
```

Comment:
this seems to work even if the `apiPath` is set in `<plugin><configuration><apiPath>`. So this is nice.

Unfortunately Intellij complains if not set in `<plugin><executions><execution><configuration>`. Wonder if there is a way to make it required and make Intellij happy.

Reply:
I guess this is a bug on IntelliJ's part. As a workaround, one could either only add a configuration to the execution or add a fake apiPath to the plugin config which will then be overridden in the execution.
But I think marking a parameter as not required and then make the plugin fail because of a missing apiPath is a bit illogical.

Reply:
funny thing is, if I add `apiPath` to the execution configuration IntelliJ complains that it is missing in the plugin configuration.

Apart from that everything seems to work. So we keep the required true.